profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/ETW/EtwEventsHandler.cpp

```diff
@@ -133,9 +133,9 @@ void EtwEventsHandler::OnConnect(HANDLE hPipe)
             {
                 _pReceiver->OnEvent(timestamp, tid, version, keyword, level, id, userDataLength, pUserData);
 
-                std::stringstream builder;
-                builder << "ETW event #" << eventsCount << " | " << keyword << " - " << id;
-                _logger->Info(builder.str());
+                //std::stringstream builder;
```

Comment:
do you want to keep this in case of Debug level ? (I have a PR to check if Debug level is enabled)
if not, we can remove this

Reply:
I don't think it is even useful at any level... I only used it during debugging session.
I'm adding an #ifdef to make it explicit